alerts/plugins/port_scan_enrichment.json

```diff
@@ -0,0 +1,13 @@
+{
+  "elasticSearchAddress": "http://127.0.0.1:9200",
+  "indicesToSearch": [
+    "events-*"
```

Comment:
If your search window is the last 24 hours, you should use the events-weekly alias, rather than events-*

Reply:
This is resolved in bd3d2b